### PR TITLE
add asciidoc filetype and update markdown filetype

### PR DESCRIPTION
**Pull request by tjdgus3537**

### CODE REVIEW

File: ignore/src/types.rs [hunk 1]
```diff
@@ -81,6 +81,7 @@ use {Error, Match};
 
 const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("agda", &["*.agda", "*.lagda"]),
+    ("asciidoc", &["*.adoc", "*.asc", "*.asciidoc"]),
```
Comment (BurntSushi) [inline ignore/src/types.rs]: Is there a shorter abbreviation we can use for this? I feel like `ad` is too short and `ascii` might be too general however, so maybe this is the best we can do.

Comment (tjdgus3537) [inline ignore/src/types.rs]: agree that

**Comment by BurntSushi** [comment]
Thanks!

**Comment by BurntSushi** [comment]
Thanks!

**Pull request merged**